python/features/check.feature

```diff
@@ -0,0 +1,85 @@
+Feature: Check example models
+
+# Alarm Clock
+
+    Scenario:  Check the alarm clock model
+        Given I have the "./alarm_clock/alarm_clock.yaml" model
+        When I check the "./alarm_clock/alarm_clock.yaml" model
+        Then I should receive a message that the check was successful
+
+    Scenario:  Check the alarm clock usecase
+        Given I have the "./alarm_clock/usecase.yaml" model
+        When I check the "./alarm_clock/usecase.yaml" model with verbose
+        Then I should receive a message that the check was successful
```

Comment:
And I think we would be able to expand the `Then` to have an `And` statement that does a check of a returned list of models being checked.

Reply:
```suggestion
        Then I should receive a message that the check of the model was successful
        And I should receive a list of evaluated models being successful
```

Something like that, because the combination of those statements provides the expected modified behavior of check with the `--verbose` flag added. Which is what these feature files are supposed to do, describe the expected behavior in a way to know what to check for in your testing.